IoTCoreDefaultApp/IoTCoreDefaultApp/Strings/ko-KR/Resources.resw

```diff
@@ -647,13 +647,13 @@
     <value>{0}({1})과(와) 연결했습니다.</value>
     <comment>Format string for message displayed when pairing attempt succeeds</comment>
   </data>
```

Comment:
Change for "BluetoothInboundRegisterationSucceededFormat" has not been added. 
Change in eng:  <value>Successfully registered for inbound pairing, supported ceremonies: {0}.</value>